swig/include/python/docs/gdal_docs.i

```diff
@@ -0,0 +1,365 @@
+// gdal.AllRegister
+%feature("docstring") GDALAllRegister "
+
+Register all known configured GDAL drivers.
+Automatically called when the :py:mod:`gdal` module is loaded.
+Does not need to be called in user code unless a driver was
+deregistered and needs to be re-registered.
+See :cpp:func:`GDALAllRegister`.
+
+See Also
+--------
+:py:func:`Driver.Register`
+";
+
+// gdal.GetCacheMax
+%feature("docstring") wrapper_GDALGetCacheMax "
+
+Get the maximum size of the block cache.
+See :cpp:func:`GDALGetCacheMax`.
+
+Returns
+-------
+int
+    maximum cache size in bytes
+";
+
+// gdal.GetCacheUsed
+%feature("docstring") wrapper_GDALGetCacheUsed "
+
+Get the number of bytes in used by the block cache.
+See :cpp:func:`GDALGetCacheUsed`.
+
+Returns
+-------
+int
+    cache size in bytes
+";
+
+// gdal.GetConfigOption
+%feature("docstring") wrapper_CPLGetConfigOption "
+
+Return the value of a configuration option.
+See :cpp:func:`CPLGetConfigOption`.
+
+Parameters
+----------
+pszKey : str
+    name of the configuration option
+pszDefault : str, optional
+    default value to return if the option has not been set
+
+Returns
+-------
+str
+
+See Also
+--------
+:py:func:`GetConfigOptions`
+:py:func:`GetThreadLocalConfigOption`
+";
+
+// gdal.GetConfigOptions
+%feature("docstring") wrapper_GetConfigOptions "
+
+Return a dictionary of currently set configuration options.
+See :cpp:func:`CPLGetConfigOptions`.
+
+Returns
+-------
+dict
+
+Examples
+--------
+>>> with gdal.config_options({'A': '3', 'B': '4'}):
+...     gdal.SetConfigOption('C', '5')
+...     gdal.GetConfigOptions()
+...
+{'C': '5', 'A': '3', 'B': '4'}
+
+See Also
+--------
+:py:func:`GetConfigOption`
+:py:func:`GetGlobalConfigOptions`
```

Comment:
@rouault, does GetGlobalConfigOptions actually exists?

Reply:
apparently not